doc/man1/openssl-smime.pod.in

```diff
@@ -245,14 +245,6 @@ used multiple times if more than one signer is required. If a message is being
 verified then the signers certificates will be written to this file if the
 verification was successful.
 
-=item B<-nocerts>
-
-Don't include signers certificate when signing.
-
-=item B<-noattr>
-
-Don't include any signed attributes when signing.
```

Comment:
Well spotted,
Maybe fix up the other ones so they are just as clear as these ones are.. There needs to at least be a comma in the -nocerts one above.